rules/composer/src/ValueObject/ChangePackage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Composer\ValueObject;
+
+use Rector\Composer\Modifier\ComposerModifier;
+use Rector\Composer\Contract\ComposerModifier\ComposerModifierInterface;
+
+/**
+ * Changes one package for another
+ * @see \Rector\Composer\Tests\ValueObject\ChangePackageTest
+ */
+final class ChangePackage implements ComposerModifierInterface
```

Comment:
Is this about rename package or change package version? Class name should tell the story

Reply:
Well, it is both. When you are changing kdyby/translation to contributte/translation you need to check package and its version too

Reply:
I see. The `ChangePackage` name is like `ChangeClass`. It could be anything :) That's why we have `RenameClassRector`, `RemoveDeadClassRector`, `PrivatizeClassRector` etc.

IMO these should be 2 different classes, one for package rename and other for version change

Reply:
I agree that we need another name of this class but imho it should does what it does now - change the single line of composer.json - package name and also set the version of new package. I think it is more user friendly like run two modifiers

Reply:
What about `ReplacePackage`?

Reply:
Sounds good!